packages/web-frontend/cypress/README.md

```diff
@@ -8,7 +8,12 @@
 
    ⚠️: In **Step 2. Install node dependencies** you need to run the commands under a Windows terminal, see the note in that section.
 
-2. Use a **[LOCAL]** `.env` file for `web-frontend`, so that it points to a local `web-config-server`
+2. Point `web-frontend` to a local instance of `web-config-server`. In `packages/web-frontend/.env`, set
```

Comment:
Presuming these instructions are targeted at running locally

Reply:
They do! But I guess it's a good idea to clarify that - will do